### ISSUE TITLE
✨2019 Q2 Planning 🎉

### ISSUE THREAD

**Issue by momack2** [ipfs/team-mgmt#902]
Hello friends!

The end of the Quarter is on the horizon and with it, it is time for us to do the **OKR Planning Dance for 2019 Q2 🕺🏽💃🏽**! Note things are a little different this quarter - with a [new two-step review process](https://docs.google.com/document/d/1zA7QKD_GvOCyfuKw5JxQ3qqRw0wKlIxsvNnd_OeHfDE/edit#) to create time for feedback on top-level project priorities from stakeholders. Keep an 👁 out for that top-level brief later this week, along with notes from the review with stakeholders!

@daviddias has done a great job [explaining how this happens in the past](https://github.com/ipfs/team-mgmt/issues/698) and drafted our [Notes on OKR Planning](https://github.com/ipfs/team-mgmt/blob/master/OKR/README.md#notes-on-okr-planning), which is a great primer on how and why we use OKRs.

The TLDR is:

- [x] [Score your 2019 Q1 OKRs with End of the Quarter Scores](https://docs.google.com/spreadsheets/d/1BtOfd7s9oYO5iKsIorCpsm4QuQoIsoZzSz7GItE-9ys/edit?ts=5c2f3d49#gid=1681757723) _(I know some things are continuing to land - feel free to put draft grades that you update as the quarter finalizes)_
- If you are a Working Group Captain/TPM
  - [x] Create a PR against your Working Group doc -- https://github.com/ipfs/team-mgmt/tree/master/OKR -- to get the conversation started. See this past example to give you an idea (https://github.com/ipfs/team-mgmt/pull/799)
  - [x] Invite the Working Group members and the community to contribute to the discussion.
  - [x] Use your Working Group Roadmap to seed the conversation https://github.com/ipfs/roadmap - and update it if needed!
- [x] Kick off an Async Retrospective for your Working Group following the [Template](https://docs.google.com/document/d/1cpMAsP6giMvGR_jaBS4nzdqx2GCOvwOjTbOhLAEPkGI/edit)
- [x] Once the OKRs are ready, push them to the [IPFS 2019 Q2 OKR Spreadsheet](https://docs.google.com/spreadsheets/d/1YSeyWqXh3ImanRrTkYQHHkCofiORn68bYqM_KTLBlsA/edit#gid=1681757723)

If you are new to this whole process, check in with your Working Group colleagues and go through the [issue that covers how we did it for 2019 Q1](https://github.com/ipfs/team-mgmt/issues/792), it will provide you plenty of context. Please post all your remaining questions so that we make sure to get them answered and improve our docs in the process ✨

**Comment by momack2** [comment]
Open OKR Planning PRs:

- Development and Maintenance:
  - JS Core Team
  - Go Core Team
  - Cluster
  - GUI
  - Infrastructure
- Efforts to Support Specific Uses:
  - Integration with Web Browsers
  - Dynamic Data and Capabilities
  - Decentralized Data Stewardship
  - Local Offline Collaboration
- Community
- Project WG

**Comment by momack2** [comment]
Thanks @lidel and @hsanjuan for getting on top of this! @scout @alanshaw @olizilla @meiqimichelle @jimpick and @mikeal - please create similar issues for your teams!

**Comment by jimpick** [comment]
Our OKRs are now moved to the spreadsheet, ready for comments. 

https://docs.google.com/spreadsheets/d/1YSeyWqXh3ImanRrTkYQHHkCofiORn68bYqM_KTLBlsA/edit#gid=27675624

**Comment by momack2** [comment]
Hello lovely @ipfs/wg-captains!

We're a couple weeks into the quarter, but since most group are only just freezing their OKRs, I figure it's not too late to get everyone (or a delegate of their choice) to record a very short ~3-5min video presenting the awesome things they're planning to work on this quarter and why. For reference, check out [this great template](https://drive.google.com/file/d/10tnQZ4EoA3wDTeWt1VcBLvPmEzt0wbCL/view?usp=sharing) from Dietrich in the Web Browsers group! The awesome details in the [js-ipfs OKR list](https://github.com/ipfs/team-mgmt/pull/927) are also a great example of the explanation and rhetoric that will get the community pumped and informed - **win-win**!

The aim for these videos is to upload them all to the IPFS YouTube in a playlist (and link from wherever else seems useful) so everyone has a quick way to index into what each group is taking on for the quarter. Let's upload them to [this drive folder](https://drive.google.com/drive/folders/1SWauTkN0jETGsNmH7fzGJPLLp2AL_-TK) so we can put them all up together! We'll also be sending them to shipped@ so more collaborators can see what we have planned. Sound awesome? HECK YEAH! 🙌 🎉

Video Todo List:
- [ ] JS Core Team
- [ ] Go Core Team
- [ ] Cluster
- [ ] GUI
- [ ] Infrastructure
- [ ] Package Managers
- [x] [Integration with Web Browsers](https://drive.google.com/file/d/10tnQZ4EoA3wDTeWt1VcBLvPmEzt0wbCL/view?usp=sharing) (@autonome)
- [ ] Dynamic Data and Capabilities
- [ ] Project WG (@momack2)

**Comment by momack2** [comment]
Alright! I recorded the latest update on our Q2 Project WG OKRs, and also took a pass at a Q1 recap and Q2 overview to intro this playlist when we upload it to Youtube. They're both in the folder: https://drive.google.com/drive/folders/1SWauTkN0jETGsNmH7fzGJPLLp2AL_-TK

Looking forward to seeing everyone else's OKR videos! No high production value needed - just give enough context that it's clear _why_ we are prioritizing this work. Bonus points got tangible "help wanted" items that viewers can jump in to help with.

**Comment by jimpick** [comment]
I missed this last week ... I'll be sure to record a DDC video on Tuesday.

**Comment by jimpick** [comment]
I recorded a video for DDC with @aschmahmann and uploaded it.

**Comment by momack2** [comment]
Remaining videos:
-   JS Core Team (@alanshaw)
-   Go Core Team (@eingenito)
-   Cluster (@hsanjuan)
-   GUI (@olizilla)
-   Infrastructure (@scout)
-   Package Managers (@achingbrain)

I'm going to upload the videos we have on Monday after the weekly meeting. Would love to have a few more to go with!

**Comment by alanshaw** [comment]
Done. Apologies for the delay.

**Comment by momack2** [comment]
Thanks to everyone who has made their video so far. THEY ARE SO AWESOME! I can just imagine how valuable they will be for prospective contributors and our active users who want to know what they can expect from us. This is exactly that! I'm uploading what we have to YouTube as we speak, but would love to add in additional videos as soon as they're ready! If folks don't have time personally, feel free to empower a teammate to step up. =]

**Comment by eingenito** [comment]
Just uploaded the one for the go-ipfs WG. Sorry for the delay.

**Comment by momack2** [comment]
This was posted to YouTube and tweeted out (getting really good attention from our community - up to 87 views!) 

Thanks all for the work (and if you have more ping here and stick in the folder, or upload to the playlist yourself).

**Comment by azappella** [comment]
Hi @momack2, I was following your OKR planning process, but could not open the google documents listed on this page (e.g. https://docs.google.com/document/d/1zA7QKD_GvOCyfuKw5JxQ3qqRw0wKlIxsvNnd_OeHfDE/edit#). Would it be possible to have those with read permission? Thanks!

**Comment by momack2** [comment]
The Q2 OKR changes came from the feeling that WG Captains were getting direction too late in the OKR-writing process, and that we weren't charting our course as a unified project. Most things stayed the same, but we also created a high level picture of the project's priorities and goals for the quarter, which we reviewed and agreed upon before each WG started drafting OKRs. 

These goals for Q2 were:
1. **P0: The IPFS protocol and its implementations are ready to support Package Manager Requirements** 
The IPFS Protocol must meet certain benchmarks on load and speed to support package manager experimentation and adoption without any major blockers.
2. **P0: Package Managers can easily understand how to use IPFS - and there are live examples of them working in the wild**
The IPFS Protocol must be sufficiently well documented, communicated, and usable to support experimentation and adoption by package manager experts less familiar with distributed web tools/primitives
3. **P1: IPFS Camp is extremely successful at growing project engagement and usability**
We need to prepare content, review applications, and handle coordination for IPFS Camp
4. **P2: The IPFS Team becomes lean, minimizes overhead in planning, and improves org structure and tools**
The community feels reengaged and aware of where the project is going and where people are spending their time, and there are org tools to plan efficiently.
5. **P3: IPFS supports current and future partners to increase adoption of the distributed web**
There are strategic projects we should take on this quarter to support our community and grow adoption of IPFS - including Filecoin’s upcoming testnet launch.

Doc now publicly accessible - feel free to dive deeper there: https://docs.google.com/document/d/11w8hP9OFbMYebuhCDFF63r3zjvur0dslgQnAhqtBgpo/edit#
